final_prep/annotations.rb

```diff
@@ -19,25 +25,36 @@ def build_a_bear(name, age, fur, clothes, special_power)
   return built_bear
 end
 
+# Two instances of build_a_bear class: Fluffy and Sleepy
```

Comment:
Let's be careful here - `build_a_bear` is a method but _not_ a class, so these aren't instances, they are just calls to the method. In a sentence, you might say "on line 29, I am calling the `build_a_bear` method".